build_all.sh

```diff
@@ -10,6 +10,7 @@ set -eux
 
 target=${target:-"NULL"}
 compiler=${compiler:-"intel"}
+PW_CSP=${PW_CSP:-} # TODO: This is an implementation from EPIC and consistent with the UFS WM build system.
```

Comment:
Can you add a comment defining PW_CSP and what the valid choices are?

Reply:
Good morning, George. The `PW_CSP` is defined, at login, by `/etc/profile.d/pw.sh`. In hindsight, the user should need to know or change anything here.

Would you still like me to add the suggested comment?

Reply:
Ok. I thought the user might have to edit this variable. Since that is not the case, you don't need to include a comment.